node/Cargo.toml

```diff
@@ -94,7 +94,7 @@ sp-timestamp = { default-features = false, git = 'https://github.com/compound-fi
 frame-system = { git = 'https://github.com/compound-finance/substrate', branch = 'master' }
 
 [features]
-default = ['with-parity-db']
+default = ['with-rocks-db', 'runtime-benchmarks']
```

Comment:
benchmarking w rocks and running parity is a bit of a mismatch, but had to do it to get it to run. apparently benchmarking on parity is still wip

Reply:
I don't think these should be defaults, right? We should move `'with-rocks-db'` under `runtime-benchmarks = [` on line 100 and remove both of these from defaults.

Reply:
yes, my mistake